filament/src/materials/colorGrading/colorGrading.mat

```diff
@@ -124,27 +122,28 @@ vec3 bloom(const vec3 color) {
     return result;
 }
 
-vec4 resolveFragment(const ivec2 uv) {
+vec4 resolveFragment(const highp vec2 uv) {
 #if POST_PROCESS_OPAQUE
-    return vec4(texelFetch(materialParams_colorBuffer, uv, 0).rgb, 1.0);
+    return vec4(textureLod(materialParams_colorBuffer, uv, 0.0).rgb, 1.0);
```

Comment:
Why are you going from texelFetch to textureLod here? Isn't texelFetch what we want?